solvebio/test/test_object.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+# from solvebio.resource import Vault
+
+from .helper import SolveBioTestCase
+
+
+class ObjectTests(SolveBioTestCase):
+
+    def test_object_paths(self):
+        vaults = self.client.Vault.all()
+        for vault in vaults:
+            for file_ in list(vault.ls().solve_objects())[:5]:
+                o_path, _ = self.client.Object.validate_path(file_.full_path)
+                self.assertEqual(o_path, file_.full_path)
+
+    def test_object_path_cases(self):
+
+        user = self.client.User.retrieve()
+        domain = user.account.domain
+        user_vault = '{0}:user-{1}'.format(domain, user.id)
+        test_cases = [
+            ['{0}:myVault'.format(domain), '{0}:myVault:/'.format(domain)],
+            ['acme:myVault', 'acme:myVault:/'],
+            ['myVault', '{0}:/myVault'.format(user_vault)],
```

Comment:
I *think* we may want to raise an error in this case since it is pretty ambiguous...

Reply:
See here: https://github.com/solvebio/api.solvebio.com/pull/1583/files#diff-5e6e2bc0f160dfe279ad0a8638bc0ba0R63